.github/workflows/check-catalyst.yaml

```diff
@@ -29,28 +35,31 @@ jobs:
 
       - name: Build Catalyst-Runtime
         run: |
-          # TODO: reenable kokkos builds
           COMPILER_LAUNCHER="" \
+          C_COMPILER=$(which ${{ needs.constants.outputs[format('c_compiler.{0}', matrix.compiler)] }}) \
+          CXX_COMPILER=$(which ${{ needs.constants.outputs[format('cxx_compiler.{0}', matrix.compiler)] }}) \
           RT_BUILD_DIR="$(pwd)/runtime-build" \
-          ENABLE_LIGHTNING_KOKKOS=OFF \
-          CMAKE_ARGS="-DENABLE_SANITIZER=ON" \
+          ENABLE_LIGHTNING_KOKKOS=ON \
           ENABLE_OPENQASM=ON \
+          ENABLE_ASAN=OFF \
```

Comment:
Why, the `CMAKE_ARGS` flag, wasn't able to build the runtime with ASAN?

Reply:
This is more convenient :)

Reply:
Sounds good! I thought that you reported something about not properly building the runtime with ASAN on our actions on Friday and assumed there was an issue with this `CMAKE_ARGS` setup.